src/ci/bin/test.inc.sh

```diff
@@ -557,11 +564,23 @@ cromwell::private::setup_prior_version_resources() {
     else
       prior_version=$((current_version - 1))
     fi
+    echo "${prior_version}"
+}
+
+cromwell::private::get_prior_version_config() {
+    local prior_version=$1
+    prior_config="${CROMWELL_BUILD_RESOURCES_DIRECTORY}/${CROMWELL_BUILD_BACKEND_TYPE}_${prior_version}_application.conf"
+    echo ${prior_config}
+}
+
+cromwell::private::setup_prior_version_resources() {
+    local prior_config
+    local prior_version=$(cromwell::private::calculate_prior_version_tag)
```

Comment:
Ditto on quotes question here. (Noting that I _have_ run into places that should not be quoted, so these are questions!)

```suggestion
    local prior_version="$(cromwell::private::calculate_prior_version_tag)"
```